ethcore/src/trace/types/error.rs

```diff
@@ -82,8 +90,9 @@ impl fmt::Display for Error {
 			BadInstruction => "Bad instruction",
 			StackUnderflow => "Stack underflow",
 			OutOfStack => "Out of stack",
-			BuiltIn => "Built-in failed",
-			Wasm => "Wasm runtime error",
+			SubStackUnderflow => "Subroutine stack underflow",
+			OutOfSubStack => "Subroutine stack overflow",			BuiltIn => "Built-in failed",
+			InvalidSubEntry => "Invalid subroutine entry",			Wasm => "Wasm runtime error",
```

Comment:
Is this intended to be on same line without line break?

Reply:
Oops! I'll fix it